config/routes.rb

```diff
@@ -1,7 +1,18 @@
 Rails.application.routes.draw do
   root 'reviews#show'
-  resources :reviews, only: [:show, :create]
+
+  resources :reviews,       only: [:show, :create]
   resources :cards
+  resources :users,         only: [:new, :create, :edit, :update]
+  resources :sessions, only: [:new, :create, :destroy]
+
+  get '/sign_up', to: 'users#new', as: 'sign_up'
+  get '/log_in', to: 'sessions#new', as: :log_in
+  delete '/log_out', to: 'sessions#destroy', as: :log_out
+
+  post 'oauth/callback', to: 'oauths#callback'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.